### PR TITLE
[CN-159] cluster connection

### PR DESCRIPTION
**Pull request by SeriyBg**
Add flag for switching cluster config creation on and off with default value set to `true`

### CODE REVIEW

File: stable/hazelcast-enterprise/values.yaml [hunk 1]
```diff
@@ -437,5 +437,10 @@ mancenter:
     # failureThreshold is the minimum consecutive failures for the probe to be considered failed after having succeeded
     failureThreshold: 3
 
+  # Cluster config creation will create the connection to the Hazelcast cluster based on the yaml.hazelcast-client
+  clusterConfigCreation: 
+    # enabled is a flag to used to enable cluster config creation
+    enabled: true
```
Comment (hasancelik) [inline stable/hazelcast-enterprise/values.yaml:443]: `clusterConfigCreation` is a little bit longer, I think. Also we should follow the same convention with other parameters like `service` , `serviceAccount` and `rbac`:
https://github.com/hazelcast/charts/blob/master/stable/hazelcast/values.yaml#L146
https://github.com/hazelcast/charts/blob/master/stable/hazelcast/values.yaml#L172
https://github.com/hazelcast/charts/blob/master/stable/hazelcast/values.yaml#L167
```suggestion
  clusterConfig:
    create: true
```

**Review by hasancelik — CHANGES_REQUESTED** [review]

**Review by hasancelik — APPROVED** [review]
LGTM, thanks for the PR 👍

**Review by alparslanavci — APPROVED** [review]
Added a small comment, otherwise LGTM 👍

File: stable/hazelcast/README.md [hunk 2]
```diff
@@ -172,6 +172,7 @@ The following table lists the configurable parameters of the Hazelcast chart and
 | mancenter.ingress.hosts | List of hostnames for ingress, see values.yaml for example| []|
 | mancenter.ingress.tls | List of TLS configuration for ingress, see values.yaml for example| []|
 | mancenter.secretsMountName| Secret name that is mounted as '/secrets/' (e.g. with keystore/trustore files) | nil |
+| mancenter.clusterConfig.create|Cluster config creation will create the connection to the Hazelcast cluster based on the yaml.hazelcast-client|true|
```
[the remark this answers is not in the document]
Comment (SeriyBg) [reply stable/hazelcast/README.md]: I've meant the parameter in the helm chart `yaml: hazelcast-client`, but seems that it causes confusion. But now I see that in other places it's called mentioned as `hazelcast-client.yaml`, so have changed it accordingly

**Pull request merged**